[Automation] java run fails for org.apache.thrift:libthrift:0.17.0

## java run on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="0.17.0" ./gradlew clean javaTest -Pcoordinates="org.apache.thrift:libthrift:0.13.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25248087844/job/74035700545
Last 300 lines of the log:
```console

> Task :clean

> Task :tckTest
====================
Testing library: org.apache.thrift:libthrift:0.13.0
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :jar
> Task :compileTestJava
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

TSSLTransportFactoryTest > sslContextLoadsTrustStoreFromContextClassLoaderResource() FAILED
    java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
        at org.apache.thrift.transport.TSSLTransportFactory.<clinit>(TSSLTransportFactory.java:46)
        at org_apache_thrift.libthrift.TSSLTransportFactoryTest.sslContextLoadsTrustStoreFromContextClassLoaderResource(TSSLTransportFactoryTest.java:40)

        Caused by:
        java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
            at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
            at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
            at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
            ... 2 more

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

4 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2s
14 actionable tasks: 4 executed, 10 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.apache.thrift:libthrift:0.13.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 0.17.0 failed with exit code 1.
SOFT-FAIL[native-image run][0.17.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="0.17.0" ./gradlew clean test -Pcoordinates="org.apache.thrift:libthrift:0.13.0"]
========================================================================================
 org.apache.thrift:libthrift:0.13.0:0.17.0 stage javac compile
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 799ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileTestJava

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 936ms
10 actionable tasks: 1 executed, 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
========================================================================================
 org.apache.thrift:libthrift:0.13.0:0.17.0 stage java run
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 759ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

TSSLTransportFactoryTest > sslContextLoadsTrustStoreFromContextClassLoaderResource() FAILED
    java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
        at org.apache.thrift.transport.TSSLTransportFactory.<clinit>(TSSLTransportFactory.java:46)
        at org_apache_thrift.libthrift.TSSLTransportFactoryTest.sslContextLoadsTrustStoreFromContextClassLoaderResource(TSSLTransportFactoryTest.java:40)

        Caused by:
        java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
            at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
            at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
            at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
            ... 2 more

4 tests completed, 1 failed

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
13 actionable tasks: 3 executed, 10 up-to-date

BUILD FAILED in 2s

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'java run' for 0.17.0 failed with exit code 1.
FAILED[java run][0.17.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="0.17.0" ./gradlew clean javaTest -Pcoordinates="org.apache.thrift:libthrift:0.13.0"]
```

## java run on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="0.17.0" ./gradlew clean javaTest -Pcoordinates="org.apache.thrift:libthrift:0.13.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25248087844/job/74035700567
Last 300 lines of the log:
```console

> Task :clean

> Task :tckTest
====================
Testing library: org.apache.thrift:libthrift:0.13.0
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :jar
> Task :compileTestJava
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

TSSLTransportFactoryTest > sslContextLoadsTrustStoreFromContextClassLoaderResource() FAILED
    java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
        at org.apache.thrift.transport.TSSLTransportFactory.<clinit>(TSSLTransportFactory.java:46)
        at org_apache_thrift.libthrift.TSSLTransportFactoryTest.sslContextLoadsTrustStoreFromContextClassLoaderResource(TSSLTransportFactoryTest.java:40)

        Caused by:
        java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
            at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
            at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
            at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
            ... 2 more

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

4 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s
14 actionable tasks: 4 executed, 10 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.apache.thrift:libthrift:0.13.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 0.17.0 failed with exit code 1.
SOFT-FAIL[native-image run][0.17.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="0.17.0" ./gradlew clean test -Pcoordinates="org.apache.thrift:libthrift:0.13.0"]
========================================================================================
 org.apache.thrift:libthrift:0.13.0:0.17.0 stage javac compile
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 764ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileTestJava

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 932ms
10 actionable tasks: 1 executed, 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
========================================================================================
 org.apache.thrift:libthrift:0.13.0:0.17.0 stage java run
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 745ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :processTestResources
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :testClasses

> Task :test FAILED

TSSLTransportFactoryTest > sslContextLoadsTrustStoreFromContextClassLoaderResource() FAILED
    java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
        at org.apache.thrift.transport.TSSLTransportFactory.<clinit>(TSSLTransportFactory.java:46)
        at org_apache_thrift.libthrift.TSSLTransportFactoryTest.sslContextLoadsTrustStoreFromContextClassLoaderResource(TSSLTransportFactoryTest.java:40)

4 tests completed, 1 failed

        Caused by:
        java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
            at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
            at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
            at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
            ... 2 more

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/tests/test/index.html

* Try:
> Run with --scan
Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
 to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 1s
13 actionable tasks: 3 executed, 10 up-to-date

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'java run' for 0.17.0 failed with exit code 1.
FAILED[java run][0.17.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="0.17.0" ./gradlew clean javaTest -Pcoordinates="org.apache.thrift:libthrift:0.13.0"]
```

## java run on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="0.17.0" ./gradlew clean javaTest -Pcoordinates="org.apache.thrift:libthrift:0.13.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25248087844/job/74035700583
Last 300 lines of the log:
```console

> Task :clean

> Task :tckTest
====================
Testing library: org.apache.thrift:libthrift:0.13.0
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :jar
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

TSSLTransportFactoryTest > sslContextLoadsTrustStoreFromContextClassLoaderResource() FAILED
    java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
        at org.apache.thrift.transport.TSSLTransportFactory.<clinit>(TSSLTransportFactory.java:46)
        at org_apache_thrift.libthrift.TSSLTransportFactoryTest.sslContextLoadsTrustStoreFromContextClassLoaderResource(TSSLTransportFactoryTest.java:40)

        Caused by:
        java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
            at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
            at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
            at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
            ... 2 more

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.


4 tests completed, 1 failed

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

FAILURE: Build failed with an exception.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

14 actionable tasks: 4 executed, 10 up-to-date
BUILD FAILED in 2s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.apache.thrift:libthrift:0.13.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 0.17.0 failed with exit code 1.
SOFT-FAIL[native-image run][0.17.0][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="0.17.0" ./gradlew clean test -Pcoordinates="org.apache.thrift:libthrift:0.13.0"]
========================================================================================
 org.apache.thrift:libthrift:0.13.0:0.17.0 stage javac compile
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 802ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileTestJava

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 919ms
10 actionable tasks: 1 executed, 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
========================================================================================
 org.apache.thrift:libthrift:0.13.0:0.17.0 stage java run
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 787ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

TSSLTransportFactoryTest > sslContextLoadsTrustStoreFromContextClassLoaderResource() FAILED
    java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
        at org.apache.thrift.transport.TSSLTransportFactory.<clinit>(TSSLTransportFactory.java:46)
        at org_apache_thrift.libthrift.TSSLTransportFactoryTest.sslContextLoadsTrustStoreFromContextClassLoaderResource(TSSLTransportFactoryTest.java:40)

        Caused by:
        java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
            at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
            at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
            at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
            ... 2 more


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/problems/problems-report.html
4 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.thrift/libthrift/0.13.0/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
13 actionable tasks: 3 executed, 10 up-to-date

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'java run' for 0.17.0 failed with exit code 1.
FAILED[java run][0.17.0][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="0.17.0" ./gradlew clean javaTest -Pcoordinates="org.apache.thrift:libthrift:0.13.0"]
```

--- a/metadata/org.apache.thrift/libthrift/0.17.0/reachability-metadata.json
+++ b/metadata/org.apache.thrift/libthrift/0.17.0/reachability-metadata.json
@@ -1,248 +1,218 @@
 {
-  "reflection": [
+  "reflection" : [
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "com.sun.crypto.provider.AESCipher$General",
-      "methods": [
+      "type" : "com.sun.crypto.provider.AESCipher$General",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "com.sun.crypto.provider.ARCFOURCipher",
-      "methods": [
+      "type" : "com.sun.crypto.provider.ARCFOURCipher",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "com.sun.crypto.provider.ChaCha20Cipher$ChaCha20Poly1305",
-      "methods": [
+      "type" : "com.sun.crypto.provider.ChaCha20Cipher$ChaCha20Poly1305",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "com.sun.crypto.provider.DESCipher",
-      "methods": [
+      "type" : "com.sun.crypto.provider.DESCipher",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "com.sun.crypto.provider.DESedeCipher",
-      "methods": [
+      "type" : "com.sun.crypto.provider.DESedeCipher",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "com.sun.crypto.provider.DHParameters",
-      "methods": [
+      "type" : "com.sun.crypto.provider.DHParameters",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "com.sun.crypto.provider.GaloisCounterMode$AESGCM",
-      "methods": [
+      "type" : "com.sun.crypto.provider.GaloisCounterMode$AESGCM",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "com.sun.crypto.provider.TlsMasterSecretGenerator",
-      "methods": [
+      "type" : "com.sun.crypto.provider.TlsMasterSecretGenerator",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.TEnumHelper"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.TEnumHelper"
       },
-      "type": "java.lang.reflect.Method[]"
+      "type" : "java.lang.reflect.Method[]"
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.partial.EnumCache"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.partial.EnumCache"
       },
-      "type": "java.lang.reflect.Method[]"
+      "type" : "java.lang.reflect.Method[]"
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "java.security.AlgorithmParametersSpi"
+      "type" : "java.security.AlgorithmParametersSpi"
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "javax.net.ssl.SSLContext"
+      "type" : "javax.net.ssl.SSLContext"
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "jdk.net.LinuxSocketOptions"
+      "type" : "jdk.net.LinuxSocketOptions"
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.partial.EnumCache"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "org_apache_thrift.libthrift.EnumCacheTest$ExampleCachedEnum"
-    },
-    {
-      "condition": {
-        "typeReached": "org.apache.thrift.meta_data.FieldMetaData"
-      },
-      "type": "org_apache_thrift.libthrift.FieldMetaDataTest$GeneratedStruct"
-    },
-    {
-      "condition": {
-        "typeReached": "org.apache.thrift.TEnumHelper"
-      },
-      "type": "org_apache_thrift.libthrift.TEnumHelperTest$ExampleThriftEnum"
-    },
-    {
-      "condition": {
-        "typeReached": "org.apache.thrift.meta_data.FieldMetaData"
-      },
-      "type": "org_apache_thrift.libthrift.ThriftMetadataInnerThriftStructTest$GeneratedStruct"
-    },
-    {
-      "condition": {
-        "typeReached": "org.apache.thrift.partial.ThriftMetadata$ThriftStruct"
-      },
-      "type": "org_apache_thrift.libthrift.ThriftMetadataInnerThriftStructTest$GeneratedStruct"
-    },
-    {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
-      },
-      "type": "sun.security.provider.JavaKeyStore$DualFormatJKS",
-      "methods": [
+      "type" : "sun.security.provider.JavaKeyStore$DualFormatJKS",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "sun.security.provider.JavaKeyStore$JKS",
-      "methods": [
+      "type" : "sun.security.provider.JavaKeyStore$JKS",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "sun.security.provider.NativePRNG",
-      "methods": [
+      "type" : "sun.security.provider.NativePRNG",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": [
+          "name" : "<init>",
+          "parameterTypes" : [
             "java.security.SecureRandomParameters"
           ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "sun.security.provider.SHA",
-      "methods": [
+      "type" : "sun.security.provider.SHA",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "sun.security.ssl.SSLContextImpl$TLSContext",
-      "methods": [
+      "type" : "sun.security.ssl.SSLContextImpl$TLSContext",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type": "sun.security.ssl.TrustManagerFactoryImpl$PKIXFactory",
-      "methods": [
+      "type" : "sun.security.ssl.TrustManagerFactoryImpl$PKIXFactory",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     }
   ],
-  "resources": [
+  "resources" : [
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "glob": "META-INF/services/java.net.spi.URLStreamHandlerProvider"
+      "glob" : "META-INF/services/java.net.spi.URLStreamHandlerProvider"
     },
     {
-      "condition": {
-        "typeReached": "org.apache.thrift.partial.EnumCache"
+      "condition" : {
+        "typeReached" : "org.apache.thrift.partial.EnumCache"
       },
-      "glob": "org/slf4j/impl/StaticLoggerBinder.class"
+      "glob" : "org/slf4j/impl/StaticLoggerBinder.class"
     }
   ]
 }

--- a/metadata/org.apache.thrift/libthrift/0.17.0/reachability-metadata.json
+++ b/metadata/org.apache.thrift/libthrift/0.17.0/reachability-metadata.json
@@ -1,194 +1,248 @@
 {
-  "reflection" : [
+  "reflection": [
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "com.sun.crypto.provider.AESCipher$General",
-      "methods" : [
+      "type": "com.sun.crypto.provider.AESCipher$General",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "com.sun.crypto.provider.ARCFOURCipher",
-      "methods" : [
+      "type": "com.sun.crypto.provider.ARCFOURCipher",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "com.sun.crypto.provider.ChaCha20Cipher$ChaCha20Poly1305",
-      "methods" : [
+      "type": "com.sun.crypto.provider.ChaCha20Cipher$ChaCha20Poly1305",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "com.sun.crypto.provider.DESCipher",
-      "methods" : [
+      "type": "com.sun.crypto.provider.DESCipher",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "com.sun.crypto.provider.DESedeCipher",
-      "methods" : [
+      "type": "com.sun.crypto.provider.DESedeCipher",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "com.sun.crypto.provider.DHParameters",
-      "methods" : [
+      "type": "com.sun.crypto.provider.DHParameters",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "com.sun.crypto.provider.GaloisCounterMode$AESGCM",
-      "methods" : [
+      "type": "com.sun.crypto.provider.GaloisCounterMode$AESGCM",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "com.sun.crypto.provider.TlsMasterSecretGenerator",
-      "methods" : [
+      "type": "com.sun.crypto.provider.TlsMasterSecretGenerator",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.TEnumHelper"
       },
-      "type" : "java.security.AlgorithmParametersSpi"
+      "type": "java.lang.reflect.Method[]"
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.partial.EnumCache"
       },
-      "type" : "sun.security.provider.JavaKeyStore$DualFormatJKS",
-      "methods" : [
+      "type": "java.lang.reflect.Method[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      },
+      "type": "java.security.AlgorithmParametersSpi"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      },
+      "type": "javax.net.ssl.SSLContext"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      },
+      "type": "jdk.net.LinuxSocketOptions"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.thrift.partial.EnumCache"
+      },
+      "type": "org_apache_thrift.libthrift.EnumCacheTest$ExampleCachedEnum"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.thrift.meta_data.FieldMetaData"
+      },
+      "type": "org_apache_thrift.libthrift.FieldMetaDataTest$GeneratedStruct"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.thrift.TEnumHelper"
+      },
+      "type": "org_apache_thrift.libthrift.TEnumHelperTest$ExampleThriftEnum"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.thrift.meta_data.FieldMetaData"
+      },
+      "type": "org_apache_thrift.libthrift.ThriftMetadataInnerThriftStructTest$GeneratedStruct"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.thrift.partial.ThriftMetadata$ThriftStruct"
+      },
+      "type": "org_apache_thrift.libthrift.ThriftMetadataInnerThriftStructTest$GeneratedStruct"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
+      },
+      "type": "sun.security.provider.JavaKeyStore$DualFormatJKS",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "sun.security.provider.JavaKeyStore$JKS",
-      "methods" : [
+      "type": "sun.security.provider.JavaKeyStore$JKS",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "sun.security.provider.NativePRNG",
-      "methods" : [
+      "type": "sun.security.provider.NativePRNG",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [
+          "name": "<init>",
+          "parameterTypes": [
             "java.security.SecureRandomParameters"
           ]
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "sun.security.provider.SHA",
-      "methods" : [
+      "type": "sun.security.provider.SHA",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "sun.security.ssl.SSLContextImpl$TLSContext",
-      "methods" : [
+      "type": "sun.security.ssl.SSLContextImpl$TLSContext",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "type" : "sun.security.ssl.TrustManagerFactoryImpl$PKIXFactory",
-      "methods" : [
+      "type": "sun.security.ssl.TrustManagerFactoryImpl$PKIXFactory",
+      "methods": [
         {
-          "name" : "<init>",
-          "parameterTypes" : [ ]
+          "name": "<init>",
+          "parameterTypes": []
         }
       ]
     }
   ],
-  "resources" : [
+  "resources": [
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.transport.TSSLTransportFactory"
+      "condition": {
+        "typeReached": "org.apache.thrift.transport.TSSLTransportFactory"
       },
-      "glob" : "META-INF/services/java.net.spi.URLStreamHandlerProvider"
+      "glob": "META-INF/services/java.net.spi.URLStreamHandlerProvider"
     },
     {
-      "condition" : {
-        "typeReached" : "org.apache.thrift.partial.EnumCache"
+      "condition": {
+        "typeReached": "org.apache.thrift.partial.EnumCache"
       },
-      "glob" : "org/slf4j/impl/StaticLoggerBinder.class"
+      "glob": "org/slf4j/impl/StaticLoggerBinder.class"
     }
   ]
 }

--- a/stats/org.apache.thrift/libthrift/0.17.0/execution-metrics.json
+++ b/stats/org.apache.thrift/libthrift/0.17.0/execution-metrics.json
@@ -107,5 +107,114 @@
       "test_file": "tests/src/org.apache.thrift/libthrift/0.17.0/src/test/java/org_apache_thrift/libthrift/TSSLTransportFactoryTest.java",
       "metadata_file": "metadata/org.apache.thrift/libthrift/0.17.0/reachability-metadata.json"
     }
+  },
+  "fix_java_run_fail:2026-06-06": {
+    "timestamp": "2026-06-06T18:08:59.592021Z",
+    "library": "org.apache.thrift:libthrift:0.17.0",
+    "starting_commit": "0db76886ea08654469b86c0748be36de8f73ca30",
+    "ending_commit": "4a11ec6ae9668de4a01c3092e71fa67e59a6a56f",
+    "previous_library": "org.apache.thrift:libthrift:0.17.0",
+    "strategy_name": "java_run_iterative_with_coverage_sources_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "0.17.0",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 8,
+            "totalCalls": 8
+          },
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 9,
+        "totalCalls": 9
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 696,
+          "missed": 28584,
+          "ratio": 0.02377,
+          "total": 29280
+        },
+        "line": {
+          "covered": 182,
+          "missed": 6729,
+          "ratio": 0.026335,
+          "total": 6911
+        },
+        "method": {
+          "covered": 45,
+          "missed": 1590,
+          "ratio": 0.027523,
+          "total": 1635
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "0.17.0",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 8,
+            "totalCalls": 8
+          },
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 9,
+        "totalCalls": 9
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 696,
+          "missed": 28584,
+          "ratio": 0.02377,
+          "total": 29280
+        },
+        "line": {
+          "covered": 182,
+          "missed": 6729,
+          "ratio": 0.026335,
+          "total": 6911
+        },
+        "method": {
+          "covered": 45,
+          "missed": 1590,
+          "ratio": 0.027523,
+          "total": 1635
+        }
+      }
+    },
+    "metrics": {
+      "input_tokens_used": 25059,
+      "cached_input_tokens_used": 104448,
+      "output_tokens_used": 1474,
+      "iterations": 1,
+      "cost_usd": 0.0964,
+      "tested_library_loc": 182,
+      "metadata_entries": 35,
+      "test_only_metadata_entries": 13,
+      "previous_library_metadata_entries": 35,
+      "previous_library_test_only_metadata_entries": 13,
+      "code_coverage_percent": 2.63,
+      "previous_library_coverage_percent": 2.63
+    },
+    "artifacts": {
+      "test_file": "tests/src/org.apache.thrift/libthrift/0.17.0/src/test/java/org_apache_thrift/libthrift/TSSLTransportFactoryTest.java",
+      "metadata_file": "metadata/org.apache.thrift/libthrift/0.17.0/reachability-metadata.json"
+    }
   }
 }

--- a/tests/src/org.apache.thrift/libthrift/0.17.0/src/test/resources/META-INF/native-image/reachability-metadata.json
+++ b/tests/src/org.apache.thrift/libthrift/0.17.0/src/test/resources/META-INF/native-image/reachability-metadata.json
@@ -49,6 +49,36 @@
           "parameterTypes" : [ ]
         }
       ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.apache.thrift.partial.EnumCache"
+      },
+      "type" : "org_apache_thrift.libthrift.EnumCacheTest$ExampleCachedEnum"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.apache.thrift.meta_data.FieldMetaData"
+      },
+      "type" : "org_apache_thrift.libthrift.FieldMetaDataTest$GeneratedStruct"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.apache.thrift.TEnumHelper"
+      },
+      "type" : "org_apache_thrift.libthrift.TEnumHelperTest$ExampleThriftEnum"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.apache.thrift.meta_data.FieldMetaData"
+      },
+      "type" : "org_apache_thrift.libthrift.ThriftMetadataInnerThriftStructTest$GeneratedStruct"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.apache.thrift.partial.ThriftMetadata$ThriftStruct"
+      },
+      "type" : "org_apache_thrift.libthrift.ThriftMetadataInnerThriftStructTest$GeneratedStruct"
     }
   ]
 }

--- a/tests/src/org.apache.thrift/libthrift/0.17.0/user-code-filter.json
+++ b/tests/src/org.apache.thrift/libthrift/0.17.0/user-code-filter.json
@@ -5,6 +5,9 @@
     },
     {
       "includeClasses" : "org.apache.thrift.**"
+    },
+    {
+      "includeClasses" : "org_apache_thrift.libthrift.**"
     }
   ]
 }

